modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -96,7 +96,7 @@ public function allow_block_editor_login() {
 		$query = wp_parse_url( urldecode( $_GET['redirect_to'] ), PHP_URL_QUERY );
 		$args  = wp_parse_args( $query );
 
-		if ( $this->framing_allowed( $args['frame-nonce'] ) ) {
+		if ( ! empty( $args['frame-nonce'] ) && $this->framing_allowed( $args['frame-nonce'] ) ) {
```

Comment:
Can we move this up to with the `redirect_to` check?

Reply:
I'm not sure how, we first have to parse it out of the `redirect_to` parameter to be able to test it

Reply:
Oh! Now I see, got it. 👍